lib/membrane/video_compositor/handler/inputs_description.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Membrane.VideoCompositor.Handler.InputsDescription do
+  @moduledoc """
+  Definition of all VC input videos used in composition.
+  """
+
+  alias Membrane.Pad
+  alias Membrane.VideoCompositor.Handler.InputProperties
+
+  @typedoc """
+  Describe all VC input videos used in composition.
+  """
+  @type t :: %{Pad.ref_t() => InputProperties.t()}
```

Comment:
NIT: this module is so small that `InputProperties` can be part of it to make it easier for reading.

Reply:
Good one